vmdb/app/models/vm_or_template/operations/snapshot.rb

```diff
@@ -51,10 +51,23 @@ def raw_remove_snapshot(snapshot_id)
     end
   end
 
+  #
+  # For some types of VMs, the process for removing
+  # evm stapshots is very different from that of
+  # removing normal snapshots.
+  #
+  # Here, we differentiate between the two, so the
+  # methods can be overridden by the subclass as needed.
+  #
+
   def remove_snapshot(snapshot_id)
     raw_remove_snapshot(snapshot_id)
   end
 
+  def remove_evm_snapshot(snapshot_id)
```

Comment:
Why can't the caller call `remove_snapshot`?  Does this make the caller code clearer?